<!-- News Section ============================================- -->  
<TABLE WIDTH="100%" BGCOLOR="#888888" BORDER="0" CELLSPACING="0" CELLPADDING="1">
    <TR><TD>        
    <TABLE BGCOLOR="#fffee8" WIDTH="100%"  BORDER="0" CELLSPACING="0" CELLPADDING="0">
        <TR><TD>        
        <TABLE BGCOLOR="#D9E4F8" BACKGROUND="images/hr.gif" WIDTH="100%" BORDER=0>
            <TR><TD>
            <!--cheap trick to get background on row to be continuous-->
            <TABLE CELLPADDING=0 CELLSPACING=0 WIDTH="100%">
                <TR><TD ALIGN="left">
                 <font size="4"><a name="TOC"></a><b>&nbsp; News</b></font><a href="http://www.twitter.com/GenomeBrowser" target="_blank"><img src="http://twitter-badges.s3.amazonaws.com/t_small-a.png" title="Follow &#64;GenomeBrowser on Twitter" alt="Follow &#64;GenomeBrowser on Twitter" style="vertical-align:text-bottom; margin-left: 15px;"></a>
                 <a title="Genome Browser Facebook page"
                 href="http://www.facebook.com/ucscGenomeBrowser"
                 target="_blank"><img
                 src="/images/FB-f-Logo__blue_22.jpg"
                 style="vertical-align:text-bottom; margin-left: 15px;"
                 alt="Genome Browser Facebook page"></a>
                 <a href="https://plus.google.com/107643636314131661344" rel="publisher"></a>
               </TD>
                <TD ALIGN="right"><A HREF="goldenPath/newsarch.html"><IMG 
                SRC="images/news.jpg" ALT="" ALIGN="right" BORDER=0></A>
            </TD></TR></TABLE>
        </TD></TR></TABLE>
        <TABLE BGCOLOR="#fffee8" WIDTH="100%" CELLPADDING=0>
          <TR><TH HEIGHT=3></TH></TR>
          <TR><TD WIDTH=10></TD>
          <TD>
          <P>
          To receive announcements of new genome 
          assembly releases, new software features, updates and 
          training seminars by email, subscribe to the
          <A HREF="http://groups.google.com/a/soe.ucsc.edu/group/genome-announce?hl=en"
          TARGET=_blank>genome-announce</A> mailing list. Please see our
          <a href="http://genome.ucsc.edu/blog"
          target=_blank>blog</a> for posts about Genome Browser tools, features, projects and more.
          </p>

    <!-- start news -->

	<hr>
	<p>
        <font color="#006666" size="4"><b>12 August 2015 &mdash;
	New blog post:  How to share your UCSC screenthoughts </b></font>
	</p>
	<p>
        Do you want to save your UCSC Browser screenthoughts, or share them with a colleague,
        or make a slide for a talk, or publish them in a paper?  Saving the state of your screen
        can take two forms:  static or dynamic.  You can snap and save a picture of the screen,
        or you can share a link to an active Genome Browser.  We talk about both approaches
        <A HREF = "http://genome.ucsc.edu/blog/how-to-share-your-ucsc-screenthoughts/"
        TARGET = _BLANK>here</A>
        and discuss some of the advantages and pitfalls of each.
	</p>

	<hr>
	<p>
        <font color="#006666" size="4"><b>29 June 2015 &mdash;
	GENCODE Genes Now the Default Gene Set on the Human (GRCh38/hg38) Assembly</b></font>
	<p>
	In a move towards standardizing on a common gene set within the bioinformatics 
	community, UCSC has made the decision to adopt the GENCODE set of gene models 
	as our default gene set on the human genome assembly. Today we have released 
	the GENCODE v22 comprehensive gene set as our default gene set on human 
	genome assembly GRCh38 (hg38), replacing the previous default UCSC Genes set 
	generated by UCSC. To facilitate this transition, the new gene set employs 
	the same familiar UCSC Genes schema, using nearly all the same table names 
	and fields that have appeared in earlier versions of the UCSC set.
	</p>
	<p>
	By default, the browser displays only the transcripts tagged as &quot;basic&quot; by the 
	GENCODE Consortium. These may be found in the track labeled "GENCODE Basic" in 
	the Genes and Gene Predictions track group. However, all the transcripts in the 
	GENCODE comprehensive set are present in the tables, and may be viewed by 
	adjusting the track configuration settings for the All GENCODE super-track. 
	The most recent version of the UCSC-generated genes can still be accessed in 
	the track &quot;Old UCSC Genes&quot;.
	</p>
	<p>
	The new release has 195,178 total transcripts, compared with 104,178 in the 
	previous version. The total number of canonical genes has increased from 
	48,424 to 49,534. Comparing the new gene set with the previous version:
	</p>
	<ul>
	<li>9,459 transcripts did not change.</li>
	<li>22,088 transcripts were not carried forward to the new version.</li>
	<li>43,681 transcripts are "compatible" with those in the previous set, 
	meaning that the two transcripts show consistent splicing. In most cases, 
	the old and new transcripts differ in the lengths of their UTRs.</li>
	<li>28,950 transcripts overlap with those in the previous set, but do not 
	show consistent splicing (i.e., they contain overlapping introns with 
	differing splice sites)</li>
	</ul>
	<p>
	More details about the new GENCODE Basic track can be found on the
	<a href="http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg38&g=knownGene"> 
	GENCODE Basic track description page</a>.
	</p>

	<hr>

        <!-- start archives -->
<!--
	<hr>
	<p>
        <font color="#006666" size="4"><b>12 August 2015 &mdash;
	New blog post:  How to share your UCSC screenthoughts. </b></font>
	<p>
        We talk about saving static images and dynamic Browser sessions in a new
        <A HREF = "">blog</A> post,
        and discuss some of the advantages and pitfalls of each.
        <a href="goldenPath/newsarch.html#081215">Read more</a>.
	</p>

        <p>
        <font color="#006666" size="4"><b>29 June 2015 &mdash;
        GENCODE Genes Now Default Gene Set on the Human (GRCh38/hg38) Assembly</b>:
        </font>
        In a move towards standardizing on a common gene set within the bioinformatics 
        community, UCSC has made the decision to adopt the GENCODE set of gene models 
        as our default gene set on the human genome assembly.
        <a href="goldenPath/newsarch.html#062915">Read more</a>.
        </p>
-->

        <p>
        <font color="#006666" size="4"><b>26 June 2015 &mdash;
	      New Bonobo (panPan1) Assembly Now Available in the Genome Browser</b>:
        </font>
	We are pleased to announce the release of a Genome Browser for the May 2012
	assembly of bonobo, <em>Pan paniscus</em> (Max-Planck Institute panpan1, UCSC
  version panPan1).
	<a href="goldenPath/newsarch.html#062615">Read more</a>.
        </p>

        <font color="#006666" size="4"><b>12 June 2015 &mdash;
        Data Integrator</b>: </font>
        Have you ever wished that the Table Browser could associate your custom track items
	with some other track, while retaining the item names from both? We have released a
	new tool that can do just that, and more: the Data Integrator.
	<a href="goldenPath/newsarch.html#061215">Read more</a>.
	</p>

            </TD><TD WIDTH=15>
         </TD></TR></TABLE>
    <BR></TD></TR></TABLE>
</TD></TR></TABLE>
